src/components/HitherJobMonitor/HitherJobMonitorControl.tsx

```diff
@@ -32,13 +31,17 @@ const HitherJobMonitorControl: FunctionComponent<Props> = ({ workspaceInfo, webs
         erroredJobs: hitherJobs.filter(j => (j.status === 'error')),
     }
     const { workspaceName, feedUri } = workspaceInfo;
+    const numPending = pendingJobs.length;
```

Comment:
Now we also report the pending jobs (ie, those that have not yet been sent to the server)